test/e2e/providers/aws/aws.go

```diff
@@ -61,9 +61,9 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 // credentialPath is the file path the AWS credentials file.
 // credentialAccountID is the account name the user uses to create VM instance.
 // The credentialAccountID should exist in the AWS credentials file pointing at one specific credential.
-func newAWSProvider(openShiftClient *clusterinfo.OpenShift, credentialPath,
-	credentialAccountID, instanceType, region string) (*awsProvider, error) {
-	session, err := newSession(credentialPath, credentialAccountID, region)
+func newAWSProvider(openShiftClient *clusterinfo.OpenShift, status *config.InfrastructureStatus, credentialPath,
```

Comment:
Sorry forgot to call this out before. I suggest making this infraStatus to be explicit. `status` is too generic.